packages/injective-cosmwasm/src/types.rs

```diff
@@ -0,0 +1,157 @@
+use cosmwasm_std::{StdError, StdResult};
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, PartialOrd, Ord, Hash, JsonSchema)]
+pub struct MarketId(String);
```

Comment:
any specific reason for using unnamed field? I think it decreases readability, what do you reckon?

Reply:
It's idiomatic actually, called "unit" in Rust, basically just a type alias

Reply:
also I see that `Addr` is using the same syntax, probably that's the reason ;-)

Reply:
@kakysha what I do not like about it is accessing it like that: `self.0`